tests/TenantDatabaseManagerTest.php

```diff
@@ -78,6 +78,47 @@ public function mysql_database_can_be_created_and_deleted_using_queued_commands(
         $this->assertEmpty(DB::select("SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = '$db_name'"));
     }
 
+    /** @test */
+    public function pgsql_database_can_be_created_and_deleted()
+    {
+        if (! $this->isTravis()) {
+            $this->markTestSkipped('As to not bloat your PostgreSQL instance with test databases, this test is not run by default.');
+        }
+
+        config()->set('database.default', 'pgsql');
+
+        $db_name = 'testdatabase' . $this->randomString(10);
+        app(DatabaseManager::class)->create($db_name, 'pgsql');
+        $this->assertNotEmpty(DB::select("SELECT datname FROM pg_database WHERE datname = '$db_name'"));
```

Comment:
Is this a correct way of checking that a database exists?

Reply:
It is, but Postgres converts all DB names to lowercase while the WHERE clause does not.